test/sequential/test-net-connect-local-error.js

```diff
@@ -3,28 +3,21 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+const expectedErrorCodes = ['ECONNREFUSED', 'EADDRINUSE'];
```

Comment:
Whoops, I'm too late! :-D Anyway, the comment is fine. What I suggested is just a nit. It can be ignored.